src/app/app.component.html

```diff
@@ -8,4 +8,5 @@ <h1>Angular Router App</h1>
   </ul>
 </nav>
 The routed views render in the <router-outlet>
-<router-outlet></router-outlet> 
\ No newline at end of file
+<router-outlet></router-outlet> 
+<app-college-application></app-college-application>
```

Comment:
remove <app-college-application></app-college-application> from this file. Instead create a route for college-application and map it to CollegeApplicationComponent